### PR TITLE
update rounding script

### PR DESCRIPTION
**Pull request by meteomatics**

### CODE REVIEW

File: meteomatics/rounding.py [hunk 1]
```diff
@@ -1,47 +1,59 @@
 # -*- coding: utf-8 -*-
 
 decimal_places = {
-    "cm":       1,  # centimeters (snow height)
+    "bft":      0,  # Beauforts (wind)
+    "cm":      -1,  # centimeters (snow height, hail)
     "C":        1,  # temperature
     "d":       -1,  # days AND degrees...
     "dn":       5,  # datenums
-    "eur":      1,  # euro
```
Comment (maekke97) [inline meteomatics/rounding.py:8]: sure, that this is no longer needed?

File: meteomatics/rounding.py [hunk 2]
```diff
@@ -1,47 +1,59 @@
...
+    "grainsm3": 1,  # grains per m3 (pollen)
     "hdd":      1,  # heating degree days
     "h":        1,  # hours
     "hft":      1,  # hectofeet
     "hPa":      1,  # pressures
-    "Pa":       1,  # pressures
+    "Hz":       6,  # vorticity
+    "mHz":      3,  # vorticity
+    "uHz":      1,  # vortictiy
+    "Pa":       0,  # pressures
```
Comment (maekke97) [inline meteomatics/rounding.py:22]: Why change?

File: meteomatics/rounding.py [hunk 3]
```diff
@@ -1,47 +1,59 @@
...
-    "cm":       1,  # centimeters (snow height)
+    "bft":      0,  # Beauforts (wind)
+    "cm":      -1,  # centimeters (snow height, hail)
     "C":        1,  # temperature
     "d":       -1,  # days AND degrees...
     "dn":       5,  # datenums
-    "eur":      1,  # euro
     "F":        1,  # temperature
     "ft":       1,  # feet (ceiling height)
+    "gcm2":     4,  # total column air pollution
```
Comment (maekke97) [inline meteomatics/rounding.py:11]: is that not a bit much? Where do we have that unit?

File: meteomatics/rounding.py (hunk 1 above)
Comment (meteomatics) [reply meteomatics/rounding.py:8]: I compared it with the list in https://gitlab.meteomatics.com/LIB/mm_cpp_lib/-/blob/master/include/Meteomatics_MeteoCacheGlobals.h#L227 and it isn't included there. 

I checked the cache parameter list and couldn't find any parameter that uses the unit eur.

File: meteomatics/rounding.py (hunk 2 above)
Comment (meteomatics) [reply meteomatics/rounding.py:22]: I adjusted the values according to to https://gitlab.meteomatics.com/LIB/mm_cpp_lib/-/blob/master/include/Meteomatics_MeteoCacheGlobals.h#L227

File: meteomatics/rounding.py (hunk 3 above)
Comment (meteomatics) [reply meteomatics/rounding.py:11]: The total column parameters for example: tc_ch4:gcm2

File: meteomatics/rounding.py (hunk 1 above)
Comment (maekke97) [reply meteomatics/rounding.py:8]: I would still keep it. I know we had it in use for the MarketAtlas and there might be other Energy-Production related products that might use it.

**Comment by maekke97** [comment]
Can you bump the version?

**Pull request merged**